Pcap++/src/XdpDevice.cpp

```diff
@@ -144,6 +145,7 @@ namespace pcpp
 		pollfd pollFds[1];
 		pollFds[0] = { .fd = xsk_socket__fd(socketInfo->xsk), .events = POLLIN };
 
+		std::vector<RawPacket> receiveBuffer;
```

Comment:
Isn't it better to move inside to while loop to decrease scope?

Reply:
No, because the buffer will need to be allocated every cycle. Currently it's only allocated once, and reallocated if expansion is needed. When its expanded, subsequent cycles can reuse the expanded size as is.

Reply:
@Dimi1010 Ok thanks for explanation